components/automate-cli/pkg/verifyserver/server/api/v1/sshusercheck.go

```diff
@@ -8,6 +8,7 @@ import (
 
 func (h *Handler) CheckSshUser(c *fiber.Ctx) error {
 	req := new(models.SshUserChecksRequest)
+	c.Request().Header.Set("Content-Type", "application/json")
```

Comment:
move this to client from where api request is trigger

Reply:
Done changed its location to trigger.go